src/Command/ExportTasksCommand.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\Message\ExportTasks;
+use DateTime;
+use Flow\Parquet\ParquetFile\Compressions;
+use Flow\Parquet\ParquetFile\Schema;
+use Flow\Parquet\ParquetFile\Schema\FlatColumn;
+use Flow\Parquet\ParquetFile\Schema\NestedColumn;
+use Flow\Parquet\Writer;
+use League\Csv\Reader;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Command\LockableTrait;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Messenger\Stamp\HandledStamp;
+
+class ExportTasksCommand extends BaseExportCommand
+{
+    use LockableTrait;
+
+    public function __construct(
+        private MessageBusInterface $messageBus
+    )
+    { parent::__construct(); }
+
+    protected function configure(): void
+    {
+        $this->addOptions($this)
+            ->setName('coopcycle:export:tasks')
+            ->setDescription('Export tasks');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        if (!$this->lock()) {
+            $output->writeln('The command is already running in another process.');
+            return Command::FAILURE;
+        }
+
+        [$target, $options] = $this->parseTarget(
+            $input->getOption('target'),
+            $input->getOption('unsecure')
+        );
+
+        $envelope = $this->messageBus->dispatch(new ExportTasks(
+            $this->parseDate($input->getOption('date-start')),
+            $this->parseDate($input->getOption('date-end'))
+        ));
+
+        /** @var HandledStamp $handledStamp */
+        $handledStamp = $envelope->last(HandledStamp::class);
+        $export = $handledStamp->getResult();
+
+        switch ($input->getOption('format')) {
+            case 'parquet':
+                $export = $this->csv2parquet($export);
+                break;
+            case 'csv': break;
```

Comment:
It's actually easy to also implement CSV, as it's what you get from the export code.

Reply:
Also, with CSV, this command can also be used to send exports by email.

Reply:
> It's actually easy to also implement CSV, as it's what you get from the export code.
Yeah it's already the case, the command support CSV natively, so since there is no need to convert it into another format, the switch case branch for CSV do nothing


> Also, with CSV, this command can also be used to send exports by email.
Yep i designed the functions to be easily extensible to do so